src/activate/activate.c

```diff
@@ -426,9 +434,7 @@ static int parse_argv(int argc, char *argv[]) {
                                 }
 
                         /* Empty optargs means one empty name */
-                        r = strv_extend_strv(&arg_fdnames,
-                                             strv_isempty(names) ? STRV_MAKE("") : names,
-                                             false);
+                        r = strv_extend_strv(&arg_fdnames, strv_isempty(names) ? STRV_MAKE("") : names, false);
```

Comment:
We often keep arguments separate if a ternary operator is used to make it easier to read. I don't think this is better.

Reply:
hmm, i do not see a way to distinguish these...